rdrf/rdrf/models/proms/models.py

```diff
@@ -205,9 +205,11 @@ def clean(self):
                 self.validate_one_and_only_one_cde_exists()
 
     def validate_min_max(self):
-        if self.cde.min_value is None or self.cde.max_value is None:
-            raise ValidationError(
-                f"[{self.cde.code}] The min and max values are not properly set in CDE")
+        # the slider widget used (in surveys) for integer CDEs require min and max values
+        if self.widget_config is not None and self.widget_config.strip() != "":
+            if self.cde.min_value is None or self.cde.max_value is None:
```

Comment:
Add widget_name to config ( so the slider is called "slider" ) - check for this here 

e.g. widget_config = json.loads(widget_config_json)
widget_name = widget_config.get("widget_name", "")
if widget_name == "slider"

Reply:
The modification was done.